extra/release_tool.py

```diff
@@ -1317,6 +1376,44 @@ def trigger_jenkins_build(state, tag_avail):
         print("Failed to start build:")
         traceback.print_exc()
 
+def trigger_gitlab_build(params, extra_buildparams):
+
+    try:
+        import requests
+    except ImportError:
+        print("requests module missing, try running 'sudo pip3 install requests'.")
+        sys.exit(2)
+
+    init_gitlab_creds()
+    if not GITLAB_TOKEN:
+        raise SystemExit(GITLAB_CREDS_MISSING_ERR)
+
+    headers = {'PRIVATE-TOKEN': GITLAB_TOKEN}
+
+    # Prepare json POST data
+    # See https://docs.gitlab.com/ee/api/pipelines.html#create-a-new-pipeline
+    # mender-qa ref hardcoded to master, it will not work for PR-like refs
+    postdata = {"ref": "master", "variables": []}
```

Comment:
It should be pretty easy to make a special case for `MENDER_QA_REV` and `ref` here, don't you think? I'm using PRs with mender-qa pretty often.

Reply:
Yep, you are right. Thanks for doing this in PR #673